cmd/users/main.go

```diff
@@ -130,7 +130,7 @@ func loadConfig() config {
 func connectToDB(dbConfig postgres.Config, logger logger.Logger) *sql.DB {
 	db, err := postgres.Connect(dbConfig)
 	if err != nil {
-		logger.Error(fmt.Sprintf("Failed to connect to postgres: %s", err))
+		logger.Error(fmt.Sprintf("vi vFailed to connect to postgres: %s", err))
```

Comment:
Fix this please

Reply:
I have fixed it in last push

Reply:
Do you mean, in your next commit? Because here I only see one commit

Reply:
What about now?

Reply:
Now is ok 👍